.travis.yml

```diff
@@ -10,6 +10,12 @@ jobs:
         - 10
       script:
         - yarn run eslint
+    - stage: Static Tests
```

Comment:
You  don't need the `stage statement here, otherwise :+1:

Reply:
Why is the `stage` statement useless here?